discovery.md

```diff
@@ -0,0 +1,16 @@
+# Box-client discovery: two options
+
+## Option 1: --no-cloud
+
+When the Foxbox is built with the --no-cloud flag, discovery works as follows:
+
+* Connect the box to the router using an ethernet cable. The router can just be a local switch,
+it does not need to be connected to the internet.
+* The box announces its hostname, which has a distinguishable format, e.g. foxbox<i>.local where <i> is a number
```

Comment:
Can you please explain how is this number attributed?
Also how does the user know about this number? Is it going to be printed on the box?